ewc-rfc005-issue-legal-person-identification-data.md

```diff
@@ -31,34 +31,42 @@ The LPID issuance process is implemented by PID providers according to national
 
 4. **Status:** The status of the legal person must be verified. These states are not yet regulated, but at the very least, the legal person for which an LPID is requested must be registered at a business register.
 
-The issuance process can either start in:
-1. a wallet application, the grahical user interface of the wallet *or*
+The issuance process can be done in:
+1. a wallet application, the grahical user interface of the wallet
 2. the eService of the PID provider.
+3. or a combination of the two above
```

Comment:
As the PID provider is the only one that can issue LPIDs, the 3rd option isn't viable, but covered in point 1, as the wallet can be a straing point for an LPID request.